Assets/Scripts/Inventory/Equipable.cs

```diff
@@ -31,7 +31,7 @@ public class Equipable : Item
         /// <returns>Whether it has the given ability or not</returns>
         public bool HasAbility(EquipableAbility ability)
         {
-            if (_abilities == null)
+            if (_abilities.Count == 0)
```

Comment:
I think there's a difference between == null and .Count == 0, is that right?

Reply:
there is but in this case the abilities array is never null, instead it might be an empty array

Reply:
this is a fix suggested by david as it would have been broken before